torch/testing/_internal/triton_utils.py

```diff
@@ -2,11 +2,11 @@
 
 import unittest
 
-from torch.testing._internal.inductor_utils import HAS_CUDA, HAS_GPU
+from torch.testing._internal.inductor_utils import HAS_CUDA_AND_TRITON, HAS_GPU
 from torch.utils._triton import has_triton
 
 
-requires_cuda = unittest.skipUnless(HAS_CUDA, "requires cuda")
+requires_cuda = unittest.skipUnless(HAS_CUDA_AND_TRITON, "requires cuda")
```

Comment:
```suggestion
requires_cuda = unittest.skipUnless(HAS_CUDA_AND_TRITON, "requires cuda")
```
lol a followup change after landing this one is renaming this to requires_cuda_and_triton 😛